draft-ietf-quic-transport.md

```diff
@@ -3629,19 +3629,23 @@ The details of loss detection and congestion control are described in
 The QUIC packet size includes the QUIC header and integrity check, but not the
 UDP or IP header.
 
-Clients MUST pad any Initial packet it sends to have a QUIC packet size of at
-least 1200 octets. Sending an Initial packet of this size ensures that the
-network path supports a reasonably sized packet, and helps reduce the amplitude
-of amplification attacks caused by server responses toward an unverified client
-address.
+Clients MUST pad ensure that the first Initial packet it sends is sent in a UDP
+datagram that is at least 1200 octets. Padding the Initial packet is a good way
+to ensure this, though including a 0-RTT packet in the same datagram is also a
+good way to meet this requirement.  Sending a UDP datagram of this size ensures
+that the network path supports a reasonable Maximum Transmission Unit (MTU), and
+helps reduce the amplitude of amplification attacks caused by server responses
+toward an unverified client address.
 
-An Initial packet MAY exceed 1200 octets if the client knows that the Path
-Maximum Transmission Unit (PMTU) supports the size that it chooses.
+The datagram containing an Initial packet MAY exceed 1200 octets if the client
```

Comment:
Since we're still talking about "the first Initial packet [the client] sends," perhaps "the first" or "this" instead of "an"?